src/OrchardCore.Modules/OrchardCore.Queries/Sql/GraphQL/SqlQueryFieldTypeProvider.cs

```diff
@@ -117,6 +117,7 @@ private static FieldType BuildSchemaBasedFieldType(Query query, JsonNode querySc
                     Name = nameLower,
                     Description = description,
                     Type = typeof(StringGraphType),
+                    ResolvedType = new StringGraphType(),
```

Comment:
This should not be necessary in an `ISchemaBuilder`. Using the resolved type is only mandatory if we are adding fields later on. See https://graphql-dotnet.github.io/docs/migrations/migration8/#30-code-classlanguage-textgraphtypeinitializecode-method-is-now-called-after-initialization-is-complete
This applies to the other occurrences as well.

Reply:
removed